src/main/java/org/jabref/logic/citationstyle/CitationStyleCache.java

```diff
@@ -65,8 +65,8 @@ public void listen(EntryChangedEvent entryChangedEvent) {
          * removes the citation of the removed entry as it's not needed anymore
          */
         @Subscribe
-        public void listen(EntryRemovedEvent entryRemovedEvent) {
-            citationStyleCache.invalidate(entryRemovedEvent.getBibEntry());
+        public void listen(EntriesRemovedEvent entriesRemovedEvent) {
```

Comment:
By the way, if you see an opportunity to replace the `com.google.common.eventbus` handler by JavaFX observable listeners, please go ahead and migrate them!